src/main/java/com/salesforce/dynamodbv2/mt/mappers/MtAmazonDynamoDbStreamsBase.java

```diff
@@ -36,6 +36,18 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(MtAmazonDynamoDbStreamsBase.class);
 
+    /**
+     * Parses arn and validates that tenant context matches current thread tenant context.
+     *
+     * @param arn Arn to parse.
```

Comment:
"Arn" -> "ARN"; also, this is not a sentence, so no terminal "." (i.e., "@param arn ARN to parse")

similarly elsewhere (e.g., "@return parsed ARN" on the next line)

Reply:
I am putting periods at the end of everything, because checkstyle is making me paranoid and because we have Jeremy GC ;)

Reply:
I'll fix the javadoc in a future PR (unless you beat me to it).